Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java

```diff
@@ -176,6 +178,12 @@ protected void execute() {
             parentCraft.getFluidLocations().addAll(newFluidList);
         }
 
+        //Rotates the craft's tracked locations.
+        for (Set<TrackedLocation> locations : craft.getTrackedLocations().values()) {
+            for (TrackedLocation location : locations) {
+                location.rotate(rotation, originPoint);
```

Comment:
This looks to be the issue with the build.  `TrackedLocation#rotate(MovecraftRotate)` only takes one argument.  Looking at it, I think the proper fix is to change the `rotate` function to take the origin point as an argument rather than assume it is the craft midpoint.